raco/algebra.py

```diff
@@ -1221,14 +1221,25 @@ def __eq__(self, other):
 
 class OrderBy(UnaryOperator):
 
-    """ Logical Sort operator
-    """
+    """ Logical OrderBy operator"""
 
     def __init__(self, input=None, sort_columns=None, ascending=None):
         UnaryOperator.__init__(self, input)
         self.sort_columns = sort_columns
         self.ascending = ascending
 
+    def __eq__(self, other):
+        return UnaryOperator.__eq__(self, other) and \
+            self.sort_columns == other.sort_columns and \
+            self.ascending == other.ascending
+
+    def __repr__(self):
+        return "{op}({inp!r}, {scol!r}, {asc!r})".format(
+            op=self.opname(),
+            inp=self.input,
+            scol=list(self.sort_columns),
```

Comment:
Thanks! This makes a lot more sense now. 

And to answer the other question, I do not need `list`, I should be able to remove it.